### PR TITLE
[WEBP-4988] Add questions section custom block

### PR DESCRIPTION
**Pull request by raymove**
https://moveinc.atlassian.net/browse/WEBP-4988

### CODE REVIEW

File: exporter.json [hunk 1]
```diff
@@ -739,75 +739,87 @@
...
+            "key": "image",
+            "type": "image",
+            "label": "image"
+          }
+        ]
+      },
+      {
+        "key": "io.rdc.documentation-main.questions-section",
+        "category": "Component",
+        "title": "Questions section [WIP]",
```
Comment (jbenning97) [inline exporter.json]: According to https://github.com/victor-move/exporter-rdc-documentation/blob/master/documentation/BLOCK_CREATION.md, an `icon` is needed

File: exporter.json [hunk 2]
```diff
@@ -739,75 +739,87 @@
...
+        "description": "Add the questions section",
+        "mode": "block",
+        "properties": []
```
Comment (jbenning97) [inline exporter.json:822]: Will the content be added in another PR? 
image

**Review by jbenning97 — APPROVED** [review]

**Review by victor-move — APPROVED** [review]
